certified-connectors/ahead/apiDefinition.swagger.json

```diff
@@ -123,5 +134,19 @@
     }
   },
   "security": [],
-  "tags": []
+  "tags": [],
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://aheadintranet.com"
+    },
+    {
+      "propertyName": "Privacy Policy",
+      "propertyValue": "https://www.aheadintranet.com/en/terms-and-conditions"
```

Comment:
@tjkb please provide a valid URL for the Privacy Policy. 
![image](https://user-images.githubusercontent.com/97551789/210868616-8f5dd2c7-8dbb-4810-ae09-bfeed3a2097d.png)

Reply:
@Amjed-Ayoub  it's resolved